library/Graphite/Web/Widget/Graphs.php

```diff
@@ -193,13 +194,13 @@ public function render()
                         $result[] = $div;
                     } elseif ($limit && $renderedGraphs === $limit) {
                         $result[] = sprintf(
-                            '<input type="checkbox" id="toggle-%1$s" class="collapsible-toggle">'
+                            '<input type="checkbox" id="toggle-%1$s" class="expandable-toggle">'
                             . '<label for="toggle-%1$s" class="link-button">'
-                            . '<span class="collapsible-show">%2$s</span>'
-                            . '<span class="collapsible-hide">%3$s</span>'
+                            . '<span class="expandable-expand">%2$s</span>'
+                            . '<span class="expandable-collapse">%3$s</span>'
                             . '</label>'
-                            . '<div class="collapsible">',
-                            $view->protectId($this->getMonitoredObjectIdentifier()),
+                            . '<div class="expandable-content">',
+                            $this->getMonitoredObjectIdentifier() . '-' . $urlHash,
```

Comment:
Why are you hashing the url instead of protecting the id? Its neither more unique nor computational more efficient.

Reply:
1. It has to be less unique (otherwise it doesn't work).
2. So what?

Reply:
Since we noticed that protectId is bugged and soon fixed in the framework, please change this.